src/ResourceManager/Cdn/Commands.Cdn/Profile/NewAzureRmCdnProfile.cs

```diff
@@ -0,0 +1,107 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Management.Automation;
+using System.Net;
+using Microsoft.Azure.Commands.Cdn.Common;
+using Microsoft.Azure.Commands.Cdn.Helpers;
+using Microsoft.Azure.Commands.Cdn.Models.Profile;
+using Microsoft.Azure.Commands.Cdn.Properties;
+using Microsoft.Azure.Management.Cdn;
+using Microsoft.Azure.Management.Cdn.Models;
+using SdkSku = Microsoft.Azure.Management.Cdn.Models.Sku;
+using SdkSkuName = Microsoft.Azure.Management.Cdn.Models.SkuName;
+
+namespace Microsoft.Azure.Commands.Cdn.Profile
+{
+    /// <summary>
+    /// Defines the New-AzureRmCdnProfile cmdlet.
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmCdnProfile", ConfirmImpact = ConfirmImpact.Low), OutputType(typeof(PSProfile))]
+    public class NewAzureRmCdnProfile : AzureCdnCmdletBase
```

Comment:
Is Profile a well=known CDN concept?  Profile is overloaded int he powershell context, and if this is not a well-known concept, a different name would be preferrable